test/Interop/Cxx/static/inline-static-member-var.swift

```diff
@@ -0,0 +1,27 @@
+// Test that inline static member variables are handled properly by the ClangImporter.
+
+// RUN: %empty-directory(%t)
+// RUN: %target-build-swift %s -I %S/Inputs -o %t/statics -Xfrontend -enable-cxx-interop -Xcc -std=c++17
+// RUN: %target-codesign %t/statics
+// RUN: %target-run %t/statics 2&>1
+//
+// REQUIRES: executable_test
+
+import InlineStaticMemberVar
+import StdlibUnittest
+
+var StaticsTestSuite = TestSuite("inline-static-member-var")
+
+StaticsTestSuite.test("inline-static-member-value") {
+  expectEqual(WithInlineStaticMember.staticMember, 256)
+  //TODO needs C++ stdlib symbols, fix after apple/swift#30914 is merged.
+  // expectEqual(WithInlineStaticMember.guardedStaticMember, 42)
+}
+
+StaticsTestSuite.test("inline-static-member-mutation") {
+  expectEqual(WithInlineStaticMember.staticMember, 256)
+  WithInlineStaticMember.staticMember = 128
+  expectEqual(WithInlineStaticMember.staticMember, 128)
```

Comment:
It might make sense to do read the value for `expectEqual` through a C++ function.

Reply:
Done.